src/redux/actions/userActions.ts

```diff
@@ -0,0 +1,13 @@
+export enum UserActions {
+  ADD_USER = 'ADD_USER',
+}
+
+export type Action<T, P> = {
```

Comment:
Exporting this makes sense. If you create more actions then this should be in a file with types commonly used in redux boilerplate.